content/ru/user-manual/designer/assets.md

```diff
@@ -4,117 +4,117 @@ layout: usermanual-page.hbs
 
-## Copy and Paste between Projects
+## Копирование и вставка между проектами
 
-To copy an asset or a selection of assets between projects, select the asset(s) and right-click to bring up the context menu to select 'Copy'. You can also use the hotkey Ctrl/Cmd + C instead if the context menu is not available due to being a read-only project.
+Чтобы скопировать ресурс или выбор ресурсов между проектами, выберите ресурс(ы) и щелкните правой кнопкой мыши, чтобы вызвать контекстное меню и выбрать "Copy". Вы также можете использовать горячую клавишу Ctrl/Cmd + C, если контекстное меню недоступно из-за того, что проект доступен только для чтения.
 
 <img loading="lazy" src="/images/user-manual/editor/assets-panel/right-click-copy.png" alt="Right click copy menu" width="500">
 
-In the project that you want to copy the asset(s) to, right click in the assets panel and select 'Paste'. Ctrl/Cmd + V hotkey can be used instead.
+В проекте, в который вы хотите скопировать ресурс(ы), щелкните правой кнопкой мыши в панели ресурсов и выберите "Paste". Вы также можете использовать горячую клавишу Ctrl/Cmd + V.
 
 <img loading="lazy" src="/images/user-manual/editor/assets-panel/right-click-paste.png" alt="Right click paste menu" width="500">
 
-Copy and pasting an asset will also copy its asset dependencies too. For example, here we have a model which references two materials and they reference a set of textures.
+Копирование и вставка ассета также скопирует его зависимости. Например, у нас есть модель, которая ссылается на два материала, которые, в свою очередь, ссылается на набор текстур.
 
 <img loading="lazy" src="/images/user-manual/editor/assets-panel/copy-and-paste-model-with-dependencies.png" alt="Model example" width="100%">
 
-If you copy and paste just the model asset into a different project, those asset dependencies are copied too.
+Если вы скопируете и вставите только ассет модели в другой проект, то эти зависимости будут скопированы.
 
 <img loading="lazy" src="/images/user-manual/editor/assets-panel/pasted-reference-assets.png" alt="Pasted referenced assets" width="100%">
 
-By default, it is pasted as a flat hierarchy. If you want keep the folder structure, hold Shift when the context menu is opened and an option will appear called 'Paste (keep folders)'. This will attempt to keep the folder structure using the folder you are pasting into as the root folder.
+По умолчанию ассет вставляется в виде плоской иерархии. Если вы хотите сохранить структуру папок, удерживайте клавишу Shift при открытии контекстного меню, и появится опция 'Paste (keep folders)'. Она попытается сохранить структуру папок, используя папку, в которую вы вставляете, как корневую папку.
 
 <img loading="lazy" src="/images/user-manual/editor/assets-panel/right-click-paste-keep-folders.png" alt="Right click paste (keep folders) menu" width="500">
 
 Will result in the following where the folder structure is preserved:
```

Comment:
I wonder why this line wasn't translated.

Reply:
translated on transifex